src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = (props) => {
+  const { name, age, isMarried, sex, partnerName } = props.person;
+  let relationship;
+
+  if (isMarried) {
+    const spouse = sex === 'm' ? 'wife' : 'husband';
+
+    relationship = <p className="Person__partner">{`${partnerName} is my ${spouse}`}</p>;
```

Comment:
keep the tag separate from the text that is changing. Only store the things that actually change in your variables.

```suggestion
    relationship = `${partnerName} is my ${spouse}`};
```